### PR TITLE
#14801 - Phalcon\Helper\Arr::blackList()

### PR DESCRIPTION
**Pull request by TimurFlush**
Hello!

* Type: new feature
* Link to issue: https://github.com/phalcon/cphalcon/issues/14801

**In raising this pull request, I confirm the following:**

- [x] I have read and understood the [Contributing Guidelines](https://github.com/phalcon/cphalcon/blob/master/CONTRIBUTING.md)
- [x] I have checked that another pull request for this purpose does not exist
- [x] I wrote some tests for this PR
- [x] I have updated the relevant CHANGELOG
- [ ] I have created a PR for the [documentation](https://github.com/phalcon/docs) about this change

Thanks

### CODE REVIEW

**Comment by ruudboon** [comment]
See #14802 as well

**Comment by ruudboon** [comment]
Can you sent it to the 4.1.x branch?

**Comment by TimurFlush** [comment]
> Can you sent it to the 4.1.x branch?

Oh my god, i'm sorry
One moment please

> See #14802 as well

There's a lot of conflict.

File: CHANGELOG-4.0.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 # [4.0.3](https://github.com/phalcon/cphalcon/releases/tag/v4.0.3) (2020-01-25)
 ## Added
+- Added `Phalcon\Helper\Arr::blackList()` to exclude elements of an array by the keys obtained from the elements of a blacklist [#14801](https://github.com/phalcon/cphalcon/issues/14801) [@TimurFlush](https://github.com/TimurFlush)
```
Comment (ruudboon) [inline CHANGELOG-4.0.md]: Please add this to the CHANGELOG-4.1.md

**Review by ruudboon — CHANGES_REQUESTED** [review]

**Pull request closed without merging**